scanners/zap-advanced/.helmignore

```diff
@@ -5,7 +5,13 @@
 .DS_Store
 
 parser/
-scanner/
+# this doesn't look too good but is required so that the scanners/scripts folder is included
```

Comment:
Should this file be in the pull request?

Reply:
No it shouldn't. I will remove it from the PR